### DISCUSSION TITLE
Reference Webpage?

### DISCUSSION THREAD

**Discussion by IronCodem** [ReplAPI-it/ReplAPI.it-NodeJS#53] [Improvements/R&D] [answered]
I recommend adding a reference webpage so that newcomers don't have to look through `discussions` to find the interface.

**Comment by rayhanadev** [comment, marked as answer]
@coolcoder-js hmm, what do you mean by interface.

Also are you talking about documentation for the package because that's at https://replapi-it.js.org/ (albeit it needs to be updated quite a fair bit)

**Reply by IronCodem** [reply]
Oh, I meant #42. Sorry for the confusion!

I actually did not see that...all's good!

**Reply by rayhanadev** [reply]
Oh no problem xD. That discussion is just there to give info and so people can pitch ideas since it's still in progress (to try it turn on experimentalFeatures).
